README.md

```
@@ -1,8 +1,7 @@
-
 # SIGIL
-**Open-source, offline zero-trust encryption utility with multi-layer chaining**
+**Open-source, offline zero-trust encryption utility**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)